toolchain/check/testdata/interop/cpp/no_prelude/file_not_found.carbon

```diff
@@ -0,0 +1,25 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// AUTOUPDATE
+// TIP: To test this file alone, run:
+// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/check/testdata/interop/cpp/no_prelude/file_not_found.carbon
+// TIP: To dump output, run:
+// TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/check/testdata/interop/cpp/no_prelude/file_not_found.carbon
+
+// --- fail_cpp_file_not_found.carbon
+
+library "[[@TEST_NAME]]";
+
+// CHECK:STDERR: fail_cpp_file_not_found.carbon:[[@LINE+3]]:1: error: file 'not_found.h' couldn't be opened for reading: No such file or directory [CppInteropFileNotFound]
```

Comment:
I think the "No such file or directory" reason could move to a note here

Reply:
Done.